tests/rbd/test_rbd_namespace_same_img_diff_ns.py

```diff
@@ -0,0 +1,85 @@
+from ceph.rbd.initial_config import initial_rbd_config
+from ceph.rbd.workflows.cleanup import cleanup
+from ceph.rbd.workflows.namespace import create_namespace_and_verify
+from ceph.rbd.workflows.rbd import create_single_image
+from ceph.utils import get_node_by_id
+from cli.rbd.rbd import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_namespace_creation_default_pool(client, **kw):
+    """
+    Tests img creation with the same name in diff ns
+    Args:
+        client: rbd client obj
+        **kw: test data
+    """
+    kw["client"] = client
+
+    # for all the pools mentioned in the config perform the test
+    for pool, _ in kw["config"]["rep_pool_config"].items():
+        # namespace1 creation in pool
+        kw["namespace"] = "ns1"
+        rc_ns1 = create_namespace_and_verify(**kw)
```

Comment:
we can make it directly
```
if create_namespace_and_verify(**kw):
    log.error(f"Namespace creation {kw['namespace']} failed")
    return 1
```

Reply:
:nit: expressive coding unless building for scale

Reply:
ideally return statuses should have been true or false unless we need to express more than 2 states like for raising exceptions, but all across the lib code we have returned 0 for success and 1 for failure.

Now, in test writing this way feels failure though we are checking for success
if not create_namespace_and_verify(**kw):
    log.info("pass")